src/routes/Canvas.svelte

```diff
@@ -47,6 +47,13 @@
 
 {#if $files?.[0]}
 	<div class="completeWrap">
+		<div class="inputWrap">
+			Please input the source for your image here:
+			<input
+				bind:value={$userProvidedLink}
+				style="margin-left:10px;border-radius:5px; border-color:darkblue; color:darkblue"
```

Comment:
<img width="482" alt="Screenshot 2023-05-02 at 18 48 26" src="https://user-images.githubusercontent.com/74478295/235717876-2831591f-4ea0-451e-874c-be9054383fbf.png">

The input has some interesting styling.  Gets fixed iif you use the shorthand border property

Reply:
Fixed.